src/ResourceManager/KeyVault/documentation/upcoming-breaking-changes.md

```diff
@@ -27,6 +27,90 @@
 
 # Upcoming Breaking Changes
 
+The following cmdlets were affected this release:
+
+**Add-AzureKeyVaultCertificate**
+- The -Certificate parameter has become mandatory.
+
+**Set-AzureKeyVaultManagedStorageSasDefinition**
+- The -Parameter parameter has become optional and no longer has a position.
+
+**Set-AzureKeyVaultCertificateIssuer**
+- The -IssuerProvider has become mandatory.
+
+**Undo-AzureKeyVaultCertificateRemoval**
+- The output of this cmdlet has changed from CertificateBundle to PSKeyVaultCertificate.
+
+**Undo-AzureRmKeyVaultRemoval**
+- ResourceGroupName has been removed from the InputObject parameter set, and is instead obtained from the ResourceId.
```

Comment:
@maddieclayton nit: "...and is instead obtained from the _InputObject parameter's_ ResourceId _property_"